java-runtime-builder-app/src/main/java/com/google/cloud/runtimes/builder/buildsteps/docker/DefaultDockerfileGenerator.java

```diff
@@ -30,56 +29,55 @@
  */
 public class DefaultDockerfileGenerator implements DockerfileGenerator {
 
-  private static final String DOCKERFILE = "FROM %s\n"
-      + "ADD %s %s\n";
+  private static final String DOCKERFILE_FROM = "FROM %s\n";
+  private static final String DOCKERFILE_ADD_ARTIFACT = "ADD %s %s\n";
+  private static final String DOCKERFILE_JETTY_QUICKSTART = "RUN /scripts/jetty/quickstart.sh\n";
+
+  private static final String APP_DESTINATION = "$APP_DEST";
```

Comment:
I see. Yeah then I think it's a good idea. Can you create the corresponding PRs or issues in OpenJDK/Jetty/Tomcat?